examples/github-repo/resource-role.yaml

```diff
@@ -15,6 +15,13 @@ Resources:
             Principal:
               Service: resources.cloudformation.amazonaws.com
             Action: sts:AssumeRole
+            Condition:
```

Comment:
do we need the extra condition?

Reply:
This is pulled from cloudformation-cli 

https://github.com/aws-cloudformation/cloudformation-cli/blob/9311ae5da95a2364410b8e6649500d18bb54cfcc/src/rpdk/core/templates/resource-role.yml#L18-L24